researchapp/fixtures.yml

```diff
@@ -0,0 +1,39 @@
+---
+class: !!python/name:researchapp.models.participants.Participant
+args: {}
+---
+class: !!python/name:researchapp.models.providers.Organization
+args:
+  id: smart
+  name: Smart
+---
+class: !!python/name:researchapp.models.providers.Organization
+args:
+  id: allscripts
+  name: Allscripts
+---
+class: !!python/name:researchapp.models.providers.Api
+args:
+  url: http://smart.dev.syncfor.science:9000/api/fhir/
```

Comment:
Possibly you want to use the url as a key to link organizations to OAuth Client details? (If so,  perhaps the OAuth Client details doesn't need an additional id).